packages/teleterm/src/mainProcess/windowsManager.ts

```diff
@@ -43,6 +43,17 @@ export class WindowsManager {
     } else {
       window.loadFile(path.join(__dirname, '../renderer/index.html'));
     }
+
+    // Taken from https://github.com/electron/electron/issues/4068#issuecomment-274159726
+    // selectall was removed from menus because it doesn't make sense in our context.
+    window.webContents.on('context-menu', (_, props) => {
```

Comment:
I wasn't aware that is possible to listen to events from the `window` in the `main` process, nice!

From what I see, we usually create methods as event handlers, like this:
```typescript
window.once('close', () => {
  this.saveWindowState(window);
});
```

I'd do the same here and maybe keep `selectionContextMenu` and `inputContextMenu` inside that method?

Reply:
Ah, that approach with a method makes sense, I didn't notice it.

`selectionContextMenu` and `inputContextMenu` are completely standalone vars that are only needed in the context of this file so I wanted to define them just once. 😏

That being said, it did occur to me that maybe it's not best to do this in the main scope since it calls some Electron code when the file is loaded. I'm going to move them to the constructor so that they're called only when the `WindowsManager` is initialized.